pyseeyou/locales.py

```diff
@@ -1,23 +1,53 @@
 from __future__ import unicode_literals
 
+LOCALE_SEP = "-"
```

Comment:
Both `-` and `_` are used as locale separators. Maybe set this as a default kwarg within `lookup_closest_locale`, but allow passing through a custom separator?